### PR TITLE
M114: Add coverage-model Postgres tables to pyon services

### PR DESCRIPTION
**Pull request by caseybryant**
These changes add coverage-model Postgres tables to pyon services.  These changes must be done in concert with ion-definition changes to ensure that the table configuration exists before the pyon integration Test 1 outlined in confluence page M114 Coverage Model Integration can be successfully executed.

### CODE REVIEW

**Comment by mmeisinger** [comment]
:+1:

**Comment by mmeisinger** [comment]
I've merged ion-definitions. Can you please update your branch with updated submodule pointer?

File: pyon/datastore/datastore_common.py [hunk 1]
```diff
@@ -43,7 +45,7 @@ def __init__(self, datastore_name=None, profile=None, config=None, container=Non
 class DatastoreFactory(object):
     """Helps to create instances of datastores"""
 
-    DS_BASE = "base"    # A standalone variant
+    DS_BASE = "base"    # A standalone variant                                    of
```
Comment (mmeisinger) [inline pyon/datastore/datastore_common.py:48]: that's probably unintended

Comment (caseybryant) [inline pyon/datastore/datastore_common.py:48]: It looks like you updated the ion-definitions pointer.  Anything else need to be done here?

Comment (caseybryant) [inline pyon/datastore/datastore_common.py:48]: The of in the comment is unintended.  Should I change it and push the update?

**Comment by mmeisinger** [comment]
this PR has been merged manually

**Pull request closed without merging**